Task_TheFlexGame/index.css

```diff
@@ -1 +1,50 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/
+.container {
+    display: flex;
+    gap: 1rem;
+    padding: 25px;
+  }
+  
+  .menu {
+    height: 100%;
+    flex-basis: 20%;
+    background-color: rgb(18, 18, 145);
+    padding: 1rem;
+  }
+  .menu h2{
+    color: white;
+  }
+  .box p{
+    
+    text-align: center;
+    padding: 10px 0px;
+    
+  }
+  .card p{
+    text-align: center;
+    padding: 10px 0px;
+    height: 1px;
+  }
+  .menu .box {

```

Comment:
Uncommented background-color should be removed to avoid confusion for future developers.